lib/health_cards/health_card_constraints.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require 'base64'
+require 'zlib'
+require 'json/minify'
+
+module HealthCards
+  ## Functionality for "Health Cards are Small" section of the Smart Health Cards Specification
+  module HealthCardConstraints
+    def bundle_operations(payload)
+      bundle = payload['vc']['credentialSubject']['fhirBundle']
+      if bundle
+        entries = bundle['entry']
+        entries, url_map = redefine_uris(entries)
+
+        update_elements(entries, url_map)
+      end
+      payload
+    end
+
+    def redefine_uris(resources)
+      url_map = {}
+      resource_count = 0
+
+      resources.each do |entry|
```

Comment:
If each element of this array is an `entry`, that makes me think that the array should be called `entries` rather than `resources`.

Reply:
Changed `resources` to `entries` here and in `update_elements` method